guide/locations/_AWS.md

```diff
@@ -43,6 +43,51 @@ Below are examples of configuration options that use values specific to AWS EC2:
   For example, `securityGroups: mygroup1` or `securityGroups: [ mygroup1, mygroup2 ]`.
 
 
+### Using a Registered Key Pair
+
+You can specify a `keyPair` to use for initial provisioning as a configuration option.
+If this is omitted, Brooklyn will use jclouds to create a new ad hoc key pair at AWS
+for that machine, and it will delete it afterwards.  This is usually seamless and
+occurs behind the scenes, with the post-provision user set up and configured as normal
+for all locations.  However if you are optimizing for creation of many machines,
```

Comment:
[minor] I'm not sure the comment "However ..." is relevant here as it's not Brooklyn responsibility to prepare reliable images that work also when creating many machines at once. I'd remove it.

Reply:
good point, have cleaned this up